state/pgstatestorage/pgstatestorage.go

```diff
@@ -159,7 +159,7 @@ func (p *PostgresStorage) GetLogs(ctx context.Context, fromBlock uint64, toBlock
 	const queryFilterByBlockHash = `AND b.block_hash = $7 `
 	const queryFilterByBlockNumbers = `AND b.block_num BETWEEN $7 AND $8 `
 
-	const queryOrder = `ORDER BY b.block_num ASC, l.log_index ASC`
+	const queryOrder = `ORDER BY b.block_num ASC, r.tx_index, l.log_index ASC`
```

Comment:
is it no better to indicate explicitly that we want to sort in  ASC order the tx_index field? we indicate this for block_num and log_index but not for tx_index field